tests/integration/cairo1/compiler_bindings/test_print.py

```diff
@@ -0,0 +1,48 @@
+from pathlib import Path
+
+import pytest
+
+from tests.integration._conftest import ProtostarProjectFixture
+from tests.integration.conftest import (
+    CreateProtostarProjectFixture,
+    assert_cairo_test_cases,
+)
+from protostar.testing.test_results import PassedTestCaseResult
+
+
+@pytest.fixture(name="protostar_project", scope="function")
+def protostar_fixture(create_protostar_project: CreateProtostarProjectFixture):
+    with create_protostar_project() as protostar_project:
+        yield protostar_project
+
+
+async def test_print(protostar_project: ProtostarProjectFixture, datadir: Path):
+    testing_summary = await protostar_project.protostar.test_cairo1(
+        datadir / "print_test.cairo",
+    )
+
+    assert_cairo_test_cases(
+        testing_summary,
+        expected_passed_test_cases_names=["test_print_basic"],
+    )
+
+    expected_outputs = {
+        "test_print_basic": [
+            "original value: [448378203247] converted to a string: [hello]",
+            "original value: [1953658213] converted to a string: [true]",
+            "original value: [439721161573] converted to a string: [false]",
+            "original value: [1953658213] converted to a string: [true]",
+            "original value: [1986358889] converted to a string: [veni]",
+            "original value: [1986618473] converted to a string: [vidi]",
+            "original value: [1986618217] converted to a string: [vici]",
```

Comment:
So 3 element array generates three line print output? I think it's fine for now, but ideally I'd consider having some better printing logic that also distinguishes between some basic types.

Reply:
That's how `debug::print` from Cairo works. I think we're good for now with what we have.